modules/core/comment/modules/plan/farm_comment_plan.module

```diff
@@ -20,3 +20,22 @@ function farm_comment_plan_entity_base_field_info(EntityTypeInterface $entity_ty
 
   return $fields;
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function farm_comment_plan_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
+  // Target the module installation form.
+  if ($form_id === 'system_modules') {
```

Comment:
This is targeting the Drupal core module form (at `/admin/modules`) not the farmOS-specific modules form (at `/setup/modules`), but the PR description seems to suggest this is a fix for `/setup/modules`.